.drone.star

```diff
@@ -180,8 +177,8 @@ def getTestPipelinesNames(ctx):
   ]
 
   for runPart in range(1, config['apiTests']['numberOfParts'] + 1):
-    pipelines.append('Core-API-Tests-ocis-storage-' + runPart)
-    pipelines.append('oC10ApiTests-owncloud-storage-ocis' + runPart)
+    pipelines.append('Core-API-Tests-ocis-storage-%s' % (runPart))
+    pipelines.append('Core-API-Tests-owncloud-storage-%s' % (runPart))
```

Comment:
This code also occurs in `getCoreApiTestPipelineNames()`
That could be called, maybe something like
`pipelineNames += getCoreApiTestPipelineNames()`